tests/Api/Admin/LocalesTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Admin;
+
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class LocalesTest extends JsonApiTestCase
+{
+    /** @test */
+    public function it_gets_locales(): void
```

Comment:
```suggestion
    public function it_gets_locales_as_loged_in_administrator(): void
```
or something like this, and you should also make a test for unathorized (I assume) :slightly_smiling_face: